topics/assembly/tutorials/get-started-genome-assembly/slides.html

```diff
@@ -0,0 +1,395 @@
+- **k-mer frequency from Illumina reads** : genome size, ploidy level, GC content, heterozygosity and even repeats composition!
+]
+
+.pull-right[
+.image-20[
+  ![GenomeScope Profile](../../images/kmers-frequency.png)
+]]
+
+---
+
+### The best possible DNA
+
+.left[
+Select the best possible DNA source and extraction.
+The extraction of **high-quality DNA** is the **most important** aspect of a successful genome project
+
+The **lack of a good starting material** will limit the choice of sequencing technology and will **affect the quality of obtained data**
+]
+
+---
+
+### The best possible DNA: Chemical purity of DNA
+
+.left[
+**Sample-related contaminants:**
+- Polysaccharides
+- Proteoglycans
+- Proteins
+- Secondary metabolites
+- Polyphenols
+- Humic acids
+- Pigments
+- Etc,...
+
+All these contaminants **can impair the efficacy of library preparation** in any technology, especially true **for PCR-free libraries** (both PacBio and ONT)
+]
+
+---
+
+### The best possible DNA: Quantity of DNA
+
+.left[
+Different technologies require different amount of DNA:
+- Illumina and 10x > 3 ng
+- BioNano > 200 ng
+- ONT > 1 μg
+- Dovetail (Hic) > 5 μg
+- PacBio > 15 μg
+]
+
+---
+
+### The best possible DNA: Structural integrity of DNA
+
+.left[
+High Molecular Weight (**HMW**) for Nanopore/PacBio (obtained mainly from fresh material)
+]
+
+---
+
+### The best possible DNA: Tips
+
+.left[
+- Many *DNA extraction protocol* available for a lot of species/taxa (VGP, Darwin Tree of Life, Nanopore, PacBio, etc)
+- **Keep DNA sample** from the same individual in case of library preparation/sequencing fail, need more coverage, new sequencing technology, etc
+- Use a **single individual** and sequence a **haploid, highly inbred diploid** organism, or **isogenic** individual
+]
+
+---
+
+### Appropriate sequencing technology
+
+.left[
+Mostly **dependant of the quantity and quality of DNA and the cost** but many parameters need to be considered prior to running an NGS experiment:
+
+- Short versus long reads or both
+- Read length
+- Read quality/error rate
+- Genome read coverage/depth
+- Library preparation
+- Downstream applications
+]
+
+---
+
+### Appropriate sequencing technology: Primary assembly
+
+.left[
+- **Illumina:** short reads (up to 2x250bp) with high quality reads. Sequencing bias with AT/GC rich regions
+- **IonTorrent:** short reads (up to 500bp) with medium quality reads
+- **Nanopore:** long reads (average ~15kbp) with low quality reads. Errors are not randomly distributed!
+- **PacBio:**
+  - CLR: long reads (average ~20kbp) with low quality reads
+  - HiFi: long reads (average ~15kbp) with high quality reads
+-
+]
+
+---
+
+### Appropriate sequencing technology: Scaffolding
+
+.left[
+- **HiC:** single or double **restriction enzyme fragmentation.** Need huge amount of coverage.
+- **Omni-C: sequence-independent endonuclease**
+]
+
+---
+
+### Appropriate sequencing technology: Short vs long reads
+
+.left[
+Read length and output
```

Comment:
Perhaps this slide could be explained a bit more, and it would be nice to include the paper in which is image was published.

Reply:
also here :) ty